flink-connector-gcp-pubsub/src/main/java/org/apache/flink/connector/gcp/pubsub/sink/PubSubSinkV2Builder.java

```diff
@@ -0,0 +1,93 @@
+package org.apache.flink.connector.gcp.pubsub.sink;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.api.common.serialization.SerializationSchema;
+import org.apache.flink.connector.gcp.pubsub.sink.config.GcpPublisherConfig;
+
+import java.util.Optional;
+
+import static org.apache.flink.connector.gcp.pubsub.common.PubSubConstants.DEFAULT_FAIL_ON_ERROR;
+import static org.apache.flink.connector.gcp.pubsub.common.PubSubConstants.DEFAULT_MAXIMUM_INFLIGHT_REQUESTS;
+
+/**
+ * A builder for creating a {@link PubSubSinkV2}.
+ *
+ * <p>The builder uses the following parameters to build a {@link PubSubSinkV2}:
+ *
+ * <ul>
+ *   <li>{@link GcpPublisherConfig} for the {@link com.google.cloud.pubsub.v1.Publisher}
+ *       configuration.
+ *   <li>{@link SerializationSchema} for serializing the input data.
+ *   <li>{@code projectId} for the name of the project where the topic is located.
+ *   <li>{@code topicId} for the name of the topic to send messages to.
+ *   <li>{@code maximumInflightMessages} for the maximum number of in-flight messages.
```

Comment:
```suggestion
 *   <li>{@code maximumInflightMessages} for the maximum number of inflight messages.
```
i would suggest we use the same word everywhere rather then it's different variations